ui/imports/Theme.qml

```diff
@@ -34,7 +34,5 @@ QtObject {
     readonly property int smallPadding: 10
```

Comment:
I think this file is no longer used. I guess it was removed, but re-added with a conflict. Could you try removing it and see if it still work please 😃

Reply:
It's indeed not needed anymore!